dsc/cli.py

```diff
@@ -121,5 +125,5 @@ def deposit(ctx: click.Context) -> None:
     """Send a batch of item submissions to the DSpace Submission Service (DSS)."""
     workflow = ctx.obj["workflow"]
     logger.debug(f"Beginning submission of batch ID: {workflow.batch_id}")
-    for response in workflow.run():
-        logger.debug(response)
+    submission_results = workflow.run()
+    logger.debug(f"Results of submission: {submission_results}")
```

Comment:
Since I added a concise `logger.info` summary of the results inside `Workflow.run`, I think the full response with all of the item identifiers should be in `logger.debug` but let me know if you disagree!